### PR TITLE
implemented no search filter

### LINKED ISSUE #4: No search filter type
Implement a new filter type that allows selects without a search field.

Suggestion:
EQUAL_NO_SEARCH = 128

UI-Implementation:
https://github.com/Coflnet/hypixel-react/pull/574

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #4

### CODE REVIEW

File: Filters/FilterType.cs [hunk 1]
```diff
@@ -23,6 +23,7 @@ public enum FilterType
         /// The command is a slider or input field
         /// </summary>
         RANGE = 32,
-        TEXT = 64
+        TEXT = 64,
+        EQUAL_NO_SEARCH = 128
```
Comment (Ekwav) [inline Filters/FilterType.cs:27]: Could the new type be more generic?
It could be used to disable rich ui features of other filters as well.

Comment (Ekwav) [inline Filters/FilterType.cs:27]:
```suggestion
        EQUAL_NO_SEARCH = 129
```
Make it backward compatible so outdated UIs would still render it as drop-down

**Review by Ekwav — CHANGES_REQUESTED** [review]
I like the idea. 
But this change would break the UI.

File: Filters/FilterType.cs (hunk 1 above)
Comment (matthias-luger) [reply Filters/FilterType.cs:27]: What does changing 128 to 129 achieve? That breaks the use of flags doesnt it?
Also: Outdated UIs dont break, as the drop-down-selects are currently choosen if nothing else works

Comment (matthias-luger) [reply Filters/FilterType.cs:27]: Like SIMPLE ?

Comment (Ekwav) [reply Filters/FilterType.cs:27]: Makes sense to me

Comment (Ekwav) [reply Filters/FilterType.cs:27]: 129 is both (SIMPLE and EQUAL)
This use case is precisely why I implemented this with flags in the first place. 
that your ui isn't breaking is an implementation-detail of your UI it's not safe to assume that all other UIs/scripts using the filter system did it the same way.

**Review by Ekwav — APPROVED** [review]
LGTM!

**Pull request merged**